### PR TITLE
integer increment decrement

### PR DESCRIPTION
**Pull request by kostiakoval**

### CODE REVIEW

**Comment by kostiakoval** [comment]
@hyperoslo/ios-open-source @zenangst @vadymmarkov Merge ?

File: Source/Array+Sugar.swift [hunk 1]
```diff
@@ -10,9 +10,9 @@ extension Array {
 }
 
 func nextIndex(x: Int?) -> Int? {
-  return x.map { $0 + 1 }
+  return x?.successor()
```
Comment (vadymmarkov) [inline Source/Array+Sugar.swift:13]: @kostiakoval Nice!

**Comment by 3lvis** [comment]
community_image_1416985144

**Comment by kostiakoval** [comment]
:D

**Pull request merged**